Основная часть домашнего задания 2

Foootbaler opened the pull request.

Foootbaler committed:
Основная часть домашнего задания 2

nburunova commented on README.md:
```diff
@@ -1,39 +1,9 @@
-- В каждой секции в поле Desctiption пояснить, какие методики использовались для генерации этой группы тест-кейсов
-
-http://habrahabr.ru/post/246463/ - вот тут можно почитать, как написать хороший тест-кейс, + сами поищите информацию в интернете
+Расширенный MindMap - http://imgdepo.com/show/8683769
```
1. есть такое чувство, что ты не разобрался в функционале фидли, из твоей майнд карты совсем не понтяно, что же делает это приложение. Можешь внести основной функционал фидли в майнд мапу?
2. есть неочевидные блоки, значение которых не мешало бы раскрыть: вход или регистрация в сервисе -> всплывающее окно в левом верхнем углу - что это за всплывающее окно? зачем оно там?
   Зеленая кнопка "Get started"  - ну напиши, что это обучалка для нового пользователя. А то, что она зеленая - совсем не дает нам никакой информации о функционале. Это важно, если ты проверяешь соотвествие дизайну, но ведь цель твоих тестов не в этом.
   "Больше информации" - информации о чем?
   Переделай, пожалуйста, майнд мап

nburunova commented on README.md:
```diff
@@ -1,39 +1,9 @@
-
-http://habrahabr.ru/post/246463/ - вот тут можно почитать, как написать хороший тест-кейс, + сами поищите информацию в интернете
+Расширенный MindMap - http://imgdepo.com/show/8683769
+Тест кейсы Буланаков Александр feedly.com
```
Тут ты перепутал вход (через уже существующие учетные записи Гугл, Твитер и Фидли и на самом деле много других социалок, которые ты не рассмотрел) и регистрацию (создание нового аккаунта) - это две разные вещи.
Создай новую майнд мапу, в которой опиши веточку для регистрации и веточку для входа через социалки. 
Потом попробуй написать тесты на каждую веточку, использую техники, которые я давала, а так же позитивно и негативное тестирование. Только не мешай все в кучу, сохраняй структуру, пусть покрытие будет не полным, для начала добьемся правильного небольшого результата :)

Foootbaler replied:
@nburunova 
Для регистрации и входа существует одно одинаковое окно, так что вход и регистрация не отличаются ничем, кроме как кнопки (нужно указать логин/e-mail, пароль и нажать кнопку "Go"), и так для любого сервиса, будь то Goggle, Twitter или же их родной Feedly, а насчет комплексного переделывания MindMap это да, я строил MM для начальной страницы, не имея данных о том, что действительно предлагает сайт (думал, что именно для одной страницу нужно это проделать)

Foootbaler replied:
@nburunova 
После переделывания MindMap-а у меня возник вопрос: можно ли выбрать другой модуль для тестирования? Если да, то что можешь посоветовать? В данный момент очень нуждаюсь в твоем мнении=)

nburunova replied:
Можно :) какой тебе понятнее удобнее и любимее :) прости, что поздно ответила

Foootbaler committed:
Update README.md

Исправил пока только MindMap, потому что до конца мы не разобрались. MindMap сразу расширенный. Жду замечаний или отмашки на то, чтобы писать тест-кейсы.

nburunova commented:
Пока сойдет. Давай тесты на ф-л, ты какой выбрал?

Foootbaler committed:
New update Readme.md

Добавлены тест-кейсы

Foootbaler commented:
@nburunova 
Добавлены тест-кейсы, можешь проверять

nburunova commented:
1. Поиск и добавление новостей - я бы разделила отдельно на поиск и на добавление, но так, как сделал ты тоже можно.
2. А как же тест на отмену добавления?
   3 https://gyazo.com/d9f9c16dba9827bdb29964f7ddf8a243  - вот я тут вижу, что можно добавить кнопкой +feedly ,а у тебя в тестах только +add описано, то есть у тебя не все способы добавления протестированы. Дополнишь?
3. в представлении Grouped by feeds недоступен фильтр "Oldest first" - то есть режим представления новостей пересекается с фильтрацией, у тебя это не учтено в тестах
4. меню под шестеренкой отличается для разных групп: 
5. all: https://gyazo.com/a5a9e98559658a429ca0aa61a83be1e8
6. testing (у меня есть такой блок): https://gyazo.com/8f02d8f7b5c324f487964f77556e0767
7. сам сайт SoftwareTestinf.ru: https://gyazo.com/62ce2051eb5c62fdbe6cc913898ceeeb
   Твои тесты на представление не учитывают этой разницы? Может стоит учесть?
8. Управление режимами просмотра новостей - ты проверил, что в режиме фидли новости открываются внутри фидли, а в режиме "на сайте первоисточнике" новости открываются на сайте-первоисточнике. Я еще бы сюда добавила тест, что эти режимы переключаются между собой без ошибок, несколько раз

Foootbaler commented:
@nburunova 
Надя, добрый день! Замечания учтены, тесты добавил, саму структуру тест-кейсов немного изменил, домашка готова к проверке.
https://projectt2015.testrail.net/index.php?/suites/view/10&group_by=cases:section_id&group_order=asc

nburunova commented:
принято
